src/Symfony/Component/Validator/ConstraintViolationListInterface.php

```diff
@@ -15,6 +15,8 @@
  * A list of constraint violations.
  *
  * @author Bernhard Schussek <bschussek@gmail.com>
+ *
+ * @method bool isEmpty() Is the list empty? - not implementing it is deprecated since Symfony 4.4
```

Comment:
I suggest `whether the list is empty or not` instead of the current description.

Reply:
Just changed it if you think it's better but for me the meaning is the same 😃

Reply:
The problem is that it's formulated as a question, I don't see any problem with that but it's not too common in the code base, so let's change that so it's consistent

Reply:
Thanks!